extensions/firmware/org.eclipse.smarthome.firmware.filesystem.test/src/test/groovy/org/eclipse/smarthome/firmware/filesystem/FileSystemFirmwareProviderTest.groovy

```diff
@@ -0,0 +1,724 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.

```

Comment:
... and we don't use "openHAB UG" license headers anymore (we are using more generic ones for all but automation).